src/Cake.Common/Tools/MSBuild/MSBuildVerbosityExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Cake.Core;
+using Cake.Core.Diagnostics;
+
+namespace Cake.Common.Tools.MSBuild
+{
+    /// <summary>
+    /// Contains functionality related to MSBuild verbosity.
+    /// </summary>
+    public static class MSBuildVerbosityExtensions
+    {
+        /// <summary>
+        /// Gets the MSBuild verbosity from <see cref="Verbosity"/>.
+        /// </summary>
+        /// <param name="verbosity">The verbosity.</param>
+        /// <returns>MSBuild verbosity string.</returns>
+        public static string GetMSBuildVerbosityName(this Verbosity verbosity)
```

Comment:
Just one thought. This was `private` before, now public. And as a extension method in a namespace that is imported automatically with the alias. Intellisense will find this :smile:

Any issues with this?

Reply:
Don't think so, might even be useful 😎 but could of course internalize?